.travis.yml

```diff
@@ -1,28 +1,80 @@
+    - secure: "fX9Um/12G/zNAHCYaztnk6EQZOwf1xI5ESpSkshZ2b4LQ8/s2sV64nIcjD/yRQbZ3GxBsNKdz2VenPgizliNyPTuGheOe05bNRIZKOODOjlLEsqvwjzvXNIPrWoi5xeA2ioCwPwq38QXu0DIvLsKy72YVz9pR2oi9PK5GjsRKmKLX6DOEWehfGrp1itD1mwl91mMK0gCiahspKzww5ryOBf5J0YemWoCc4ByzSGYhjL/2B/kJOS9j6Ua9UxIIPK9Qf4M+wxQct46HqY0aVk0NogAhn38ClV+9S+cMCbiUs3DCHosx/zV3wlZj4vnCCEScnbYdB/ujyCkPv8+oK+vPomOJsE94cd+4/BQYNDJ6N2ZfG23Zs8byK6PmPQqrjSKYyta1XyrvNr23k9Cn6T1h/T+WTnaB5doLZr+j91EmLuwGRxh11zolYRjKcPSZaKtGSGuFBeJtJjDTu5dvkJx8tJEWUZumjIeUKPPkbF8USJgy6/0ls1w4N08qXYAFTQAinJkF/b4Jv2HefIV1ZRN7qEQk65NNix/7NVlhdJc0rrwgcrykBz2jxvzG1SQNBMYpg+eZIB0V7BSZz+taKLjO0ZCUQOx8vzBP96PtRnJVhIPGquyLw585qE0RsLJjnFs26q4pobnhU01NdWa9SXFAHvDRIs1qi6/FmE9EPySLyk="
+    # DATABASE_PASSWORD
+    - secure: "NW4pCoTBNO1hmvecBdEUO9Ry59oFG+dGsAiLMv7bUJ73S5HAhZwQSxWxcpMHM6m3J8bcm+9Y1AC2IKxebrHMR+bghm1SRXnor+Bgi7MjRkbe/6xOX9VmwtaNMT0d1O0d71o+xwtqygwZMW+cz0DXTd4gj5MKc0N77pSlyZcdT78Wm+KtyU8WHVihsfu7YGc0vE5BO1g51reRh1DUXfEaOuVInqLSifowtICs98em+HhJVNqYRBXqyoZwFvCWOplDKhdnopfyiB8A+vf4jP1nyCrg4UeLAxw8jEWIOcqeVPKRiUMLcTTHKO2dBABVky8bLo/PyQkH9ACme9DN+aRGmfUulrRkAqhsXKLRbxlECWc+C5gKImd+D5DXTTc3xhryUjmQ1a/UyuvdO8Q1UWhwyUN3KdefqwB5dMspEvPPpZg0+uuX2tfSdfPX0muSUOrOiqxI/S34C2PY5+TM3bBVQoNZALtQF7PLc7i6cL78t2EXyB7isUgiL7Ax7JX2XupyNndzo57KBRY+HygfmUUoY57VLCO8CyFft0TAN6YpCDS74wggBh8veXDvYAzvFmejS9HxQoJ6W5ahl7yH1tyeSHC82hfaBG7E0bmxjo6C9NRmuVa/7bnyDd0MhWKkNPNuckx0Odc8tclqg5BsUzKdP8otIEZEAbHFJema1UMAxoI="
+
+# Todo: add this at the beginning of the before_install script: - if [ "TRAVIS_PULL_REQUEST" != "false" ]; then exit 0; fi
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - yes '' | pecl install apcu redis
+  - echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - echo "extension = apcu.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+
+install:
+  - sh -c "cd api && composer install --prefer-dist --no-progress --no-suggest --no-interaction --no-scripts"
+  - bin/console doctrine:database:create --env=prod
+  - bin/console doctrine:schema:update --env=prod --force
+  - bin/console hautelook:fixtures:load
 
 before_script:
-  - sudo service mysql stop
-  - sudo service postgresql stop
-  - wget https://kubernetes-helm.storage.googleapis.com/helm-v2.6.1-linux-amd64.tar.gz
-  - tar xzf helm-v2.6.1-linux-amd64.tar.gz
   - npm install -g react-scripts
   - while sudo lsof -Pi :5432 -sTCP:LISTEN -t; do sleep 1; done
+  - sudo service mysql stop
+  - sudo service postgresql stop
+  - curl -O https://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-linux-amd64.tar.gz
+  - tar -zxvf helm-v2.7.2-linux-amd64.tar.gz
 
 script:
   - docker-compose up -d
   - linux-amd64/helm lint api/helm/api/
   - sleep 20
-  - sh -c "cd admin && PUBLIC_URL="/api-demo-admin-test" yarn install && yarn build"
-  - sh -c "cd client && PUBLIC_URL="/api-demo-client-test" yarn install && yarn build"
+  - sh -c "cd admin && PUBLIC_URL="/api-demo-admin-test" yarn install --pure-lockfile && yarn build"
```

Comment:
Should be run in the Docker image (while not mandatory, it will probably be faster outside of the container)